.github/workflows/release.yaml

```
@@ -17,21 +17,21 @@ on:
 # Configure constants for this workflow.
 env:
   # The base filename of the binary produced by `cargo build`.
-  BINARY: bevy_quickstart
-  # The name to use for the packaged application produced by this workflow.
-  PACKAGE_NAME: bevy_quickstart
-  # The itch.io page to upload to, in the format: `user-name/project-name`.
-  # Comment this out to disable.
-  ITCH_TARGET: the-bevy-flock/bevy-quickstart
-  # The organization or author that owns the rights to the game.
-  OWNER: the-bevy-flock
+  cargo_binary_name: bevy_quickstart
   # The path to the assets directory.
-  ASSETS_DIR: assets
-  # Whether packages produced by this workflow should be uploaded to the Github release.
-  UPLOAD_PACKAGES_TO_GITHUB_RELEASE: true
+  assets: assets
+  # The name to use for the app produced by this workflow.
+  app_name: bevy_quickstart
+  # The organization or author that owns the rights to the app.
```

```suggestion
  # The organization or author that owns the rights to the app. Must be either kebab-case or snake_case.
```
It's an Apple thing.

---

Maybe we _do_ need to split `app_name` into two variables, with one of them being specifically for MacOS.